### PR TITLE
Black 20.8b0

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Comment by fao89** [comment]
Alternative:
pin black - https://github.com/pulp/pulpcore/pull/866

File: pulpcore/app/models/repository.py [hunk 1]
```diff
@@ -828,7 +828,10 @@ class RepositoryVersionContentDetails(models.Model):
         (REMOVED, "removed"),
     )
 
-    count_type = models.CharField(max_length=1, choices=COUNT_TYPE_CHOICES,)
+    count_type = models.CharField(
+        max_length=1,
+        choices=COUNT_TYPE_CHOICES,
+    )
```
Comment (daviddavis) [inline pulpcore/app/models/repository.py:834]:
```suggestion
    count_type = models.CharField(max_length=1, choices=COUNT_TYPE_CHOICES)
```

Comment (fao89) [reply pulpcore/app/models/repository.py:834]: all these changes were done by black, not sure if I can change, let me try

Comment (daviddavis) [reply pulpcore/app/models/repository.py:834]: There was a change in black that it automatically breaks up lists with trailing commas. If you drop the trailing commas, that will address most of these changes.

**Comment by daviddavis** [comment]
I went through all the changes black was making and tried to make intelligent decisions: https://github.com/pulp/pulpcore/pull/870

Let me know what you think.

**Comment by fao89** [comment]
closing in favor of https://github.com/pulp/pulpcore/pull/870

**Pull request closed without merging**